src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -2972,8 +2960,7 @@ void PhaseIdealLoop::do_range_check(IdealLoopTree *loop, Node_List &old_new) {
 
       // Kill the eliminated test
       C->set_major_progress();
-      Node *kill_con = _igvn.intcon(1-flip);
-      set_ctrl(kill_con, C->root());
+      Node *kill_con = intcon(1-flip);
```

Comment:
```suggestion
      Node* kill_con = intcon(1-flip);
```
We generally now have the pointer `*` with the type. So if you touch any new code please update it ;)